src/GpsTracking.ts

```diff
@@ -27,34 +26,34 @@ export class GpsTracking {
     }, 1000);
   }
 
-  getLocation(): Promise<GpsObject[]> {
```

Comment:
I wasn't able to use this as it was, because Promises weren't defined. I just return a custom promise-like object instead.